scripts/.tests-compose.yaml

```diff
@@ -31,7 +31,7 @@ services:
     volumes:
       - work:/w
     working_dir: /w/resources/public
-    command: python -m SimpleHTTPServer 80
+    command: bash -c 'npm install -g node-static && static -a 0.0.0.0 -p 80'
```

Comment:
It is not exposed to the host (which is good) but it does need to be exposed within the container network so the PhantomJS container can pull down the page for running tests.